pkg/renew/renew_test.go

```diff
@@ -63,15 +69,6 @@ func TestValidate(t *testing.T) {
 			},
 			expErr: false,
 		},
-		"If --namespace and --all namespace specified, error": {
-			options: &Options{
-				All: true,
-			},
-			setStringFlags: []stringFlag{
-				{name: "namespace", value: "foo"},
-			},
-			expErr: true,
-		},
```

Comment:
I don't think this is really correct, as looking at the [description](https://github.com/cert-manager/cmctl/blob/c42c489dd0af74bb6a6568ea55e86cced39b4dc2/pkg/renew/renew.go#L90) for `--all`, as well as the actual `Run` function, seems to suggest that `--all` should work with `--namespace`